src/core/Directus/GraphQL/FieldsConfig.php

```diff
@@ -77,6 +77,20 @@ public function getFields()
                     $fields[$v['field']] = Types::userCollection($collection);
                     break;
                 case 'o2m':
+                    $relation = $this->getRelation('o2m', $v['collection'], $v['field']);
```

Comment:
@rijkvanzanten Hi, thanks for the work :)

There is no such method called `getRelation` in this file 😱 

This change is causing massive issues in the current release, makes the whole GraphQL endpoint not load in any way, because of the missing method...

Reply:
Raised https://github.com/directus/api/issues/1795...